unic/ucd/hangul/src/lib.rs

```diff
@@ -0,0 +1,25 @@
+// Copyright 2012-2015 The Rust Project Developers.
+// Copyright 2017 The UNIC Project Developers.
+//
+// See the COPYRIGHT file at the top-level directory of this distribution.
+//
+// Licensed under the Apache License, Version 2.0 <LICENSE-APACHE or
+// http://www.apache.org/licenses/LICENSE-2.0> or the MIT license
+// <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
+// option. This file may not be copied, modified, or distributed
+// except according to those terms.
+
+#![no_std]
+#![forbid(future_incompatible, missing_debug_implementations, unconditional_recursion, unsafe_code)]
+#![deny(bad_style, unsafe_code, unused)]
+
+extern crate unic_ucd_version;
+use unic_ucd_version::UnicodeVersion;
+
+pub mod hangul;
```

Comment:
Since `hangul` algorithms are all this component providing, let's make the module here private and expose the API via `pub use`.

Also, what do you think about renaming the public API functions to `compose_syllable()` and `decompose_syllable()`.

IMHO, it would reflect the algorithm better, and is more readable.

Reply:
Agreed, refined the APIs.